applications/plugins/SofaCarving/CarvingManager.cpp

```diff
@@ -97,8 +97,6 @@ void CarvingManager::init()
         {
             core::CollisionModel* m = models[i];
             m->getContext()->get(topoMapping);
-            if (topoMapping == NULL) continue;
```

Comment:
could you remove this change from this PR not to mix the changes. A proper PR to update the CarvingManager is indeed needed.